engine/src/test/java/org/camunda/bpm/engine/test/api/encoding/ProcessEngineCharacterEncodingTest.java

```diff
@@ -87,7 +87,6 @@ protected Task newTask() {
   protected Task newTaskWithComment(String message) {
     Task task = newTask();
     taskService.createComment(task.getId(), null, message);
-    taskService.saveTask(task);
```

Comment:
Let's leave this in here. It's not necessary, but might also reveal issues with the code if something pops up due to this.

Reply:
I will remove it from this test as it is not necessary and create a dedicated test for this scenario. It turned out that this combination is actually a problem, so we should test it.